specs/interop/fault_proof.md

```diff
@@ -0,0 +1,24 @@
+# Fault Proof
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Security Considerations](#security-considerations)
+  - [Cascading dependencies](#cascading-dependencies)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+TODO
+
+No changes to the dispute game bisection are required. The only changes required are to the fault proof program itself.
+The insight is that the fault proof program can be a superset of the state transition function.
+
+## Security Considerations
+
+TODO
```

Comment:
I'm not sure if this is the best section to add it, but would love a section somewhere in the specs that answers "how does enabling interop on a chain change what an user is trusting"? 
- is it still a single honest node, or now n honest nodes for n interop-ing chains? 
- any difference for being only the source chain or only destination chain?
- what are the implications of trusting an unsafe head of another sequencer, worst case scenario? (just a reorg?)

Reply:
Separate but somewhat relevant Q - would also love an explicit write out on what must be true about participating chains to uphold these trust assumptions
- share the same fault proof contracts? 
- share L1 implementation contracts?
- have the same block time? 
i believe @tynes you phrased this + the other ask as "sketching out more directly what is needed for consensus" :)

Reply:
This is true, there needs to be an honest validator for each chain in the full transitive dependency set. Will be sure to add this explicitly